mfhdf/test/tdatainfo.c

```diff
@@ -1755,28 +1505,14 @@ static intn test_extend_SDSs()
 
 /* Test driver for testing the public function SDgetdatainfo. */
 extern int test_datainfo() {
-    intn status;
-    int num_errs = 0;
-
     /* Output message about test being performed */
     TESTING("getting location info of data (tdatainfo.c)");
 
-    /* Test nonspecial SDSs */
-    num_errs = num_errs + test_nonspecial_SDSs();
-
-    /* Test compressed SDSs */
+    int num_errs = test_nonspecial_SDSs();
     num_errs = num_errs + test_compressed_SDSs();
-
-    /* Test chunked empty SDSs */
```

Comment:
Comment from the original code was incorrect.  Should be "Test special empty SDSs".